docs/documentation/recipes.md

```diff
@@ -99,7 +99,9 @@ myplot!(args...; kw_args...) = ...
 A specialization of `argument_names` is emitted if you have an argument list
 `(x,y,z)` provided to the recipe macro:
 
-    `argument_names(::Type{<: MyPlot}) = (:x, :y, :z)`
+```julia
+argument_names(::Type{<: MyPlot}) = (:x, :y, :z)
+```
 
 This is optional but it will allow the use of `plot_object[:x]` to
```

Comment:
Later on in the stocks example, the arguments are fetched like `plot_object.x`. Is both supported or you should the above be changed to match the example?

Reply:
I prefer `plot.x` and would like to deprecate `plot[:x]` at some point, or change its meaning...

Reply:
Should I change this in the docs to bridge the gap while you are working on the overhaul? I couldn't find where `plot.x` get's wired into `getproperty`, so I didn't feel comfortable doing the change right away. Here are some other occurences:

* https://github.com/jonas-schulze/Makie.jl/blob/patch-1/MakieCore/src/recipes.jl#L121-L122
* https://github.com/jonas-schulze/Makie.jl/blob/patch-1/MakieCore/src/recipes.jl#L126
* https://github.com/jonas-schulze/Makie.jl/blob/patch-1/docs/documentation/recipes.md?plain=1#L112

Reply:
Sorry, somehow missed your comment... Yeah I think that's a good idea!

Its here: https://github.com/JuliaPlots/Makie.jl/blob/78820311c9b45347f0937caf8b31a25bbc27dd73/MakieCore/src/attributes.jl#L189

And then getproperty uses getindex here:
https://github.com/JuliaPlots/Makie.jl/blob/78820311c9b45347f0937caf8b31a25bbc27dd73/MakieCore/src/attributes.jl#L83
(Not very straightforward to find, especially with the `Transformable` supertype)